pkg/plugin/kms_v2_server.go

```diff
@@ -59,13 +59,12 @@ func (s *KeyManagementServiceV2Server) Status(ctx context.Context, _ *kmsv2.Stat
 		encryptResponse.KeyID,
 	)
 	if err != nil {
-		klog.ErrorS(err, "failed to decrypt healthcheck call")
+		mlog.Error("failed to decrypt healthcheck call", err)
 		return nil, err
 	}
 
 	if string(decryptedText) != healthCheckPlainText {
-		err = fmt.Errorf("healthcheck failed, decrypted text does not match")
-		klog.Error(err)
+		mlog.Error("healthcheck failed, decrypted text does not match", nil)
 		return nil, err
```

Comment:
This is incorrect. `err` is no longer set which means this change is always returning nil